### PR TITLE
add numSends and starRating to fetched fields and reorganize to mimic…

### PR DESCRIPTION
**Pull request by DanielW48**
… Route fields

# Describe your changes
put new fields into fetchedRoute type and reorganize to mimic how the organization is for Route object.
also adds empty export to resolve compile error for web.

# Issue ticket number and link
#182 
[#75](https://github.com/ucf-tower-app/web/issues/75)

### CODE REVIEW

File: functions/src/index.ts [hunk 1]
```diff
@@ -54,3 +54,5 @@ exports.sendMail = functions.https.onRequest((req: any, res: any) => {
     });
   });
 });
+
+export {};
```
Comment (JakeSteinebronn) [inline functions/src/index.ts:58]: Hopefully this isn't problematic in the future

**Review by JakeSteinebronn — CHANGES_REQUESTED** [review]
won't you need numRatings as well?

File: functions/src/index.ts (hunk 1 above)
Comment (DanielW48) [reply functions/src/index.ts:58]: yeah this was giving web a compile error without it, not sure why this fixes things

**Comment by DanielW48** [comment]
> won't you need numRatings as well?

so i left that out since i think it is mostly used to calculate starRating? I guess web could display numRatings to employees as well

**Review by JakeSteinebronn — APPROVED** [review]

**Pull request merged**